specs/PreferredColorScheme.md

```diff
@@ -0,0 +1,120 @@
+Preferred Color Scheme API
+===
+
+# Background
+This API's main use is to set the preferred color scheme for WebView2's which are associated with a profile. The options are similar to Edge: match the system default color scheme, change to light color scheme, or change to dark color scheme. This sets the color scheme for WebView2 UI like dialogs, prompts, and context menus by setting the ['prefers-color-scheme'](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) [media feature](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#media_features). Websites typically query for this setting in order to set CSS themes for light/dark.
+
+Please note this API will only set the overall appearance, but not theme as defined in the Edge browser.
+For reference, in the screenshot below, this API is meant to expose the Overall Appearance section as a WebView2 API.
+
+![Edge Settings Appearance Page](https://github.com/MicrosoftEdge/WebView2Feedback/blob/api-appearance/specs/images/EdgeSettingsAppearance.png)
+
+# Description
+We propose extending `CoreWebView2Profile` to include a `PreferredColorScheme` property. This property will apply
+to any WebView2's which are associated with this profile.
+
+# Examples
+
+## C++
+
+```cpp
+wil::com_ptr<ICoreWebView2> m_webView;
+
+void ViewComponent::SetPreferredColorScheme(COREWEBVIEW2_PREFERRED_COLOR_SCHEME_KIND value)
```

Comment:
I think we don't need the extra `_KIND` at the end. So it would be `COREWEBVIEW2_PREFERRED_COLOR_SCHEME` with values `COREWEBVIEW2_PREFERRED_COLOR_SCHEME_SYSTEM`, `COREWEBVIEW2_PREFERRED_COLOR_SCHEME_DARK`, and `COREWEBVIEW2_PREFERRED_COLOR_SCHEME_LIGHT`.

Because "Light" really **is** the color scheme. If it were "Kind" then that would suggest that we have a bunch of "color scheme" objects, some of which are Light, some of which are Dark, and some of which are System.

Reply:
Good point, removed _KIND from the enum now